### ISSUE TITLE
Missing base sprite when used with Wannaroo basic Industries.

### ISSUE THREAD

**Issue by Ragin1924** [zephyris/opengfx2#109]
This might be an issue with just WRBI, but it could also be an issue with Opengfx2 that could affect other industry sets. It appears that the recycling deport in WRBI is missing it's base sprite when using Opengfx2. I'm not sure how base sets work, but I'm guessing this means Opengfx2 is missing something that normal opengfx has, which might break multiple grfs not created with it in mind.

With Opengfx
![de5ff9e477aa0a65fede107265ba7d6a](https://github.com/zephyris/opengfx2/assets/93628028/24005756-e740-424f-ac8a-d275e10bde36)

With Opengfx2
![fcfa5fda30fb9a2f410f53b0c15e3b23](https://github.com/zephyris/opengfx2/assets/93628028/293ae9ad-2602-4d81-a50f-a34b82d2c648)

I'm using jgr patch pack 0.56.2 with no other grfs loaded.

**Comment by zephyris** [comment]
Very happy to investigate! But, I can't find where to download WBI(!), doesn't seem to be on BaNaNaS, no download link on the Github or forum posts?

**Comment by zephyris** [comment]
Found it, have to search for WRBI (WBI and Wannaroo don't find it!).

The recycling plant uses sprite number 2210 as a ground tile. That's normally a construction stage for the paper mill, drawn as a building sprite on the normal industry ground tile (2022). So, it's using a building sprite as a ground sprite.

In OpenGFX1 sprite 2210 happens to be a ground sprite-sized tarmack tile. In the original base graphics it is bare earth with construction framework. In OpenGFX2 it's an empty sprite. So it'll look a bit glitchy in anything except OpenGFX1.

The building (sprite 2095) poking through the fence glitch will happen with original base graphics too, the OpenGFX1 sprite is smaller. I also noticed the recycling depot uses a Toyland ground sprite (screenie below). In OpenGFX1 toyland isn't very toyland, so it looks kinda like a normal ground sprite, but in OpenGFX2 and original base graphics it'll look bad.

![image](https://github.com/zephyris/opengfx2/assets/2762690/a0afa216-8b26-441d-9ad1-6671dae540ba)

So, IMO, WRBI bugs...
